components/Card.tsx

```
@@ -8,6 +7,11 @@ import ActionsDropdown from "./ActionsDropdown";
 const Card = ({ file }: { file: FileItem }) => {
   const fileHref = file.downloadUrl || file.url || "#";
 
+  // Show a subtle message only for files still being processed after the first 2 minutes
+  const isStaleProcessing =
+    (file.aiStatus === "pending" || file.aiStatus === "processing") &&
+    Date.now() - new Date(file.createdAt).getTime() > 2 * 60 * 1000;
```

ESLint

[error] Error: Cannot call impure function during render

`Date.now` is an impure function. Calling an impure function can produce unstable results that update unpredictably when the component happens to re-render. (https://react.dev/reference/rules/components-and-hooks-must-be-pure#components-and-hooks-must-be-idempotent).

/home/jailuser/git/components/Card.tsx:13:5
  11 |   const isStaleProcessing =
  12 |     (file.aiStatus === "pending" || file.aiStatus === "processing") &&
> 13 |     Date.now() - new Date(file.createdAt).getTime() > 2 * 60 * 1000;
     |     ^^^^^^^^^^ Cannot call impure function
  14 |
  15 |   return (
  16 |     <Link href={fileHref} target="_blank" className="file-card">

(react-hooks/purity)